Server/EVESharp.Node/Services/Graphics/beyonce.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using EVESharp.EVE.Network.Services;
+using EVESharp.EVE.Network.Services.Validators;
+using EVESharp.EVE.Sessions;
+using EVESharp.Types;
+using EVESharp.Types.Collections;
+
+namespace EVESharp.Node.Services.Graphics
```

Comment:
A few of the comments made before apply to this file too, so I won't be repeating them everywhere.